### DISCUSSION TITLE
Getting issue while creating the index.

### DISCUSSION THREAD

**Discussion by tv-ankur** [supabase/supabase#13923] [Questions]
Hi all.,

I am trying to create the index but getting the following error - 

"An error has occurred: Failed to fetch"

Code to creating the index-

```
create index on public.<table_name>
using ivfflat (<column_name> vector_cosine_ops)
with (lists = 100);
```

It was earlier working but not now. Table have more than 500K rows if it makes any difference?

**Comment by prm-yash** [comment]
I was running into the same issue and tried many things but ultimately had to upgrade the plan of my Supabase project to have more compute power by getting this add-on: https://supabase.com/docs/guides/platform/compute-add-ons. 

For reference, I had ~350K rows in my database and got the XL plan. This will likely work well for your use case too, but you can take a look at these benchmarks to decide: https://supabase.com/docs/guides/ai/choosing-compute-addon
